region/maridia/outer/Fish Tank.json

```diff
@@ -891,21 +891,43 @@
       "entranceCondition": {
         "comeInRunning": {
           "speedBooster": true,
-          "minTiles": 0.4375
+          "minTiles": 4.4375
```

Comment:
I guess maybe this should have a shinecharge with like 25 tiles as well, to indicate that you tap?